Scripts/Tracking/Velocity/XRNodeVelocityEstimator.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace VRTK.Core.Tracking.Velocity
+{
+    using System.Collections.Generic;
+    using UnityEngine;
+    using UnityEngine.XR;
+
+    /// <summary>
+    /// Retrieves the velocity estimations for the specified <see cref="XRNode"/> device.
+    /// </summary>
+    public class XRNodeVelocityEstimator : VelocityTracker
+    {
+        /// <summary>
+        /// The device to query velocity estimations for.
+        /// </summary>
+        [Tooltip("The device to query velocity estimations for.")]
+        public XRNode device = XRNode.LeftHand;
+
+        /// <inheritdoc />
+        public override bool IsActive()
+        {
+            return isActiveAndEnabled;
```

Comment:
Move this into the base class instead (and update other subclasses of that if necessary).

Reply:
I'll do this in a separate commit as it will affect many other velocity trackers and doesn't feel right doing it here.